### PR TITLE
Adding rewind and fastforward - fixes for Android - breaks web.

### PR DESCRIPTION
**Pull request by annzimmer**

### CODE REVIEW

File: lib/player_widget.dart [hunk 1]
```diff
@@ -202,14 +212,14 @@ class _PlayerWidgetState extends State<PlayerWidget> {
...
         });
       }
     });
 
     _audioPlayer.onNotificationPlayerStateChanged.listen((state) {
       if (mounted) {
-        setState(() => _audioPlayerState = state);
+        setState(() => _playerState = state);
       }
     });
```
Comment (yoyomyo) [inline lib/player_widget.dart]: remove _playingRouteState below if this variable is no longer used

File: lib/player_widget.dart [hunk 2]
```diff
@@ -93,8 +90,14 @@ class _PlayerWidgetState extends State<PlayerWidget> {
                           return;
                         }
                         final Position = v * duration.inMilliseconds;
-                        _audioPlayer
-                            .seek(Duration(milliseconds: Position.round()));
+                        if (kIsWeb) {
```
Comment (yoyomyo) [inline lib/player_widget.dart]: When I tried to revert this part, scrubbing the seekbar started to work for Web

File: lib/player_widget.dart (hunk 1 above)
Comment (annzimmer) [reply lib/player_widget.dart]: yeah noticed that this morning too :)

File: lib/player_widget.dart (hunk 2 above)
Comment (annzimmer) [reply lib/player_widget.dart]: That works, thanks

File: lib/player_widget.dart [hunk 3]
```diff
@@ -255,13 +255,9 @@ class _PlayerWidgetState extends State<PlayerWidget> {
     return result;
   }
 
-
   Future<int> _rewind() async {
     Duration _tempPosition = Duration(seconds: _position!.inSeconds - 10);
```
Comment (yoyomyo) [inline lib/player_widget.dart]: max(0, _position!.inSeconds - 10)

Comment (annzimmer) [reply lib/player_widget.dart]: that's one of my edge cases I was working on next :) added min to forward as well.

**Review by yoyomyo — APPROVED** [review]

**Pull request merged**